test/C/src/federated/DistributedCountDecentralized.lf

```diff
@@ -18,10 +18,10 @@ reactor Print {
         interval_t elapsed_time = get_elapsed_logical_time();
         printf("At tag (%lld, %u), received %d. "
                "The original intended tag of the message was (%lld, %u).\n",
-                elapsed_time,
+                (long long int)elapsed_time,
```

Comment:
Is this cast needed? If so, we have *many* places that will need it.  Maybe there is a printf code for int64?

Reply:
Yes. Unfortunately, `int64_t` on my 64-bit machines is a `long int`, not a `long long int`. In this case, `%ld` would be fine on most 64-bit machines. 

However, there is a syntax for printing `int64_t` ([source](https://stackoverflow.com/questions/9225567/how-to-print-a-int64-t-type-in-c), [documentation](https://en.cppreference.com/w/cpp/types/integer)):
```
#include <inttypes.h>
int64_t my_int = 999999999999999999;
printf("%" PRId64 "\n", my_int);
```

I wouldn't expect the end user to know this. Maybe we should revert back to `long long int` for `instant_t`/`interval_t`? This would mostly be a problem for the RTI, I think, which can be resolved by some careful casting. The `long long int` specification states an integer that is "at least" 64-bit, which can cause unexpected behavior I'm sure in the future. However, 128-bit machines are probably a few years away.

Reply:
I have create an issue (#426) to track this.